lib/Newsletter/Renderer/StylesHelper.php

```diff
@@ -147,7 +149,18 @@ public static function applyLineHeight($style, $selector) {
     if (!preg_match('/mailpoet_paragraph|h[1-4]/i', $selector)) return $style;
     $lineHeight = isset($style['lineHeight']) ? (float)$style['lineHeight'] : self::$defaultLineHeight;
     $fontSize = (int)$style['fontSize'];
+    $msoLineHeight = round($lineHeight * $fontSize);
+    if ($msoLineHeight % 2 === 1) {
+      $msoLineHeight++;
+    }
+    $msoFontSize = $fontSize;
+    if ($msoFontSize % 2 === 1) {
+      $msoFontSize++;
+    }
+    $style['msoLineHeightAlt'] = sprintf('%spx', $msoLineHeight);
+    $style = ['msoFontSize' => sprintf('%spx', $msoFontSize)] + $style;
```

Comment:
Why did you use the union operator between arrays instead of setting the value for the specific key?

Reply:
It is because I wanted to achieve a specific order. This array is later used in a foreach, and the final CSS is built from it. I wanted the order of the CSS rules to match the [original issue](https://github.com/mailpoet/mailpoet/issues/3348). I don't know if that is important or not.

Reply:
Just a note, the order of CSS rules inside a single selector/`style=""` attribute doesn't change anything. Only the specificity of a selector and selector's position can change which rule wins.

Reply:
We don't use `style=""` here, we are using normal CSS declaration. And here is used the normal fontSize with the vendor-prefixed one and I thought the order is important for those.

That being said. I am not sure if it would make any difference with a different order. I tried to match what the original issue "specified".